.node-version

```diff
@@ -0,0 +1 @@
+16.18.1
```

Comment:
it can differ from the version in Dockerfile. (`node:latest`) is it fine?

Reply:
We currently do not have a use for the Dockerfile, and this file (`.node-version`) is actually for the Cloudflare Pages. As Cloudflare Pages does not support node 18.x and the default was 12.x, I had to provide this file. Although the target of the new code is 18.x, I'm pretty sure that there should be no problem running this on Cloudflare Pages (tested,) and as I'm thinking of migrating away from Cloudflare Pages in favor of Vercel Deploy, I think it's an acceptable compromise to have 16.18.1 (the lastest available LTS on Cloudflare Pages) on this file.

Reply:
OK... then, how about specifying this version to `Dockerfile` too? (or removing `Dockerfile` is also fair enough if we don't need 😅 ).

Reply:
I think I'll be removing Dockerfile, as we don't acutally use it for the moment. We can restore it in the future if we find a use 😋

Reply:
🙆‍♂️